functions/upload_postprocessing.py

```diff
@@ -83,11 +83,15 @@ def _derive_files_from_upload(trial_id: str, upload_type: str, session):
         # Save to GCS
         blob = upload_to_data_bucket(artifact.object_url, artifact.data)
 
+        # Build basic facet group
+        facet_group = f"{artifact.data_format}|{artifact.file_type}"
```

Comment:
Dunno but for my eye looks too succinct, I'd add spaces...

Reply:
This is just the internal representation (equivalent function to prism-generated `facet_group`s). The UI currently uses the same, but it doesn't need to.